app/Utils/ClientPortal/CustomMessage/CustomMessage.php

```diff
@@ -82,12 +82,12 @@ private function compose(): array
             '$company.name' => optional($this->company)->getSetting('name'),
             '$company.website' => optional($this->company)->getSetting('website'),
 
-            '$client.id' => optional(optional($this->client)->present())->id,
-            '$client.name' => optional(optional($this->client)->present())->name,
-            '$client.website' => optional(optional($this->client)->present())->website,
-            '$client.public_notes' => optional(optional($this->client)->present())->public_notes,
-            '$client.phone' => optional(optional($this->client)->present())->phone,
-            '$client.balance' => optional(optional($this->client)->present())->balance,
+            '$client.id' => optional($this->client)->id,
```

Comment:
It may help to look at the clientpresenter class, and include those methods as variable names, as they are likely to be needed here..

ie. 
client.address
client.shipping_address
client.email

etc

Reply:
Updated :+1:

Reply:
Thanks, this looks great